package.json

```diff
@@ -5,19 +5,20 @@
     "type": "git",
     "url": "https://github.com/mobify/split-test"
   },
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Hey Shawn, is this really just a minor change? Thinking what if somebody was referencing `split-test/dist/split-test.min.js`?